easybuild/easyblocks/m/molpro.py

```diff
@@ -67,8 +89,12 @@ def configure_step(self):
                     raise EasyBuildError("Failed to create symlink for license token at %s", self.license_token)
 
             else:
-                # no license token available, so fail early
-                raise EasyBuildError("No license token found at either %s or via 'license_file'", self.license_token)
+                self.cfg['license_token_present'] = False
+                self.log.warning("No licence token found at either {0} or via 'license_file'".format(self.license_token))
+        
+        # Precompiled binaries
+        if self.cfg['precompiled_binaries']:
+            return
```

Comment:
I'd rather see you indent code than `return` in the middle of a method, it makes the code harder to grasp